assets/js/modules/analytics/datastore/properties.js

```diff
@@ -476,6 +476,10 @@ const baseSelectors = {
 	getPropertiesIncludingGA4: createRegistrySelector( ( select ) => ( state, accountID ) => {
 		let properties = select( STORE_NAME ).getProperties( accountID );
 
+		if ( ! properties ) {
+			return undefined;
+		}
```

Comment:
This looks like an oversite on my behalf!  I also think that below it should return `undefined` if `propertiesGA4` is `undefined`  (AFAIK we cascade our `undefined` loading state like this)

Reply:
To elaborate, without this fix the selector crashes as it calls `properties.concat` below. Instead I could just pass an array as a default argument, but doing this seems to fit more with what we do with loading